src/pages/Question.js

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from "react";
+import Button3 from "../components/ui/Button3";
+import { nanoid } from "nanoid";
+import _ from "lodash";
+
+
+const Question = () => {
+    const [serverData, setServerData] = useState([])
+    const [session, setSession] = useState(0)
+    const [correct, setCorrect] = useState(0)
+    const [submit, setSubmit] = useState (false)
+
+    
+  
+    useEffect(()=>{
+        async function newQuestion() {
+            try {
+                let url = "https://opentdb.com/api.php?amount=5"
+                let res = await fetch(url);
+                const data = await res.json()
+                let QtnsArr = []
+                data.results.forEach(qtn =>
+                     QtnsArr.push({id: nanoid(), answers:_.shuffle([qtn.correct_answer, ...qtn.incorrect_answers]), question : qtn.question, correct:qtn.correct_answer, checked: false, selected:null}))
+                setServerData(QtnsArr)
+            } catch (error) {
+                console.log(error);
+            }
+        }
+        newQuestion();
+        
+    },[session])
+
+    function handlePlayAgain(){
+        setSession(prevSession=>prevSession +1)
```

Comment:
You should use ES6 here too.
```suggestion
  const handlePlayAgain = () => {
       
```